konduit-serving-api/src/main/java/ai/konduit/serving/util/PortUtils.java

```diff
@@ -0,0 +1,23 @@
+package ai.konduit.serving.util;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+import java.net.ServerSocket;
+
+@Slf4j
```

Comment:
We don't use slf4j here I believe? Were we supposed to be?

Reply:
Ah yeah sorry about that. There were a couple of other functions which were used to check if the ports were in use or not. Didn't end up needing them. They were using the logs

Reply:
https://github.com/KonduitAI/konduit-serving/pull/254/commits/f4f54cfaf15f3b81c17a2e7ce4a7897998986067